torch/_library/autograd.py

```diff
@@ -0,0 +1,37 @@
+from typing import Any, Callable
+
+from .. import _C, autograd
+
+
+def make_autograd_impl(opdef: Any) -> Callable:
+    def autograd_impl(*args):
+        name: str = "GeneratedBackwardFor" + opdef._name
```

Comment:
I think you want to use both namespace and name here?
Also CamelCase them?

Reply:
> I think you want to use both namespace and name here?

yup

> Also CamelCase them?

This will make it more difficult to read